drizzlepac/alignimages.py

```diff
@@ -1169,7 +1206,16 @@ def interpret_fit_rms(tweakwcs_output, reference_catalog):
                     'generate_source_catalogs() generate the .reg region files for every chip of every input image and '
                     'should generate_astrometric_catalog() generate file "refcatalog.cat"? Unless explicitly set, the '
                     'default is "False".')
-    'Should the code generate '
+
+    PARSER.add_argument( '-r', '--regtest', required=False,choices=['True','False'],default='False',help='Flag to '
+                    'accommodate use of PyTest to run regression tests.  When using PyTest with individual images '
+                    'which are part of an association (e.g., *_flt.fits), the files cannot be obtained from MAST '
+                    'as singletons. They are stored in Artifactory or local disk as part of the regression test suite. The '
+                    'default is "False".')
+
+    PARSER.add_argument( '-t', '--raw_test_data_path', required=False,default=None,help='A comma separated list of strings '
+                    'defining the path on Artifactory or local disk where the *_fl[t|c].fits files are stored.  The root of '
+                    'the path is defined by the environment variables TEST_BIGDATA. ["hst-hla-pipeline","dev","base_tests"].')
 
     ARGS = PARSER.parse_args()
```

Comment:
Why so many variables have either `hungarianNotation` or are `ALLCAPS`?

Reply:
Just a comment to the hungarianNotation or ALLCAPS...
This is the way the software engineer did this in other projects.  This code is being pulled out into its own file and being reworked.

Reply:
Issue addressed in PR #298